### DISCUSSION TITLE
Smooth out Sensor Data?

### DISCUSSION THREAD

**Discussion by pcace** [HomeSpan/HomeSpan#295] [Q&A] [answered]
Hey, i have a couple Temperature Sensors running here, and i am wondering what the best way would be to smooth out the sensor values (they are very "jumpy")...

Thanks a lot for suggestions!

**Comment by HomeSpan** [comment, marked as answer]
Exponential smoothing works well in these situations. You can implement this in the loop() method of your Service.

**Reply by pcace** [reply]
Thanks! using this library: 
https://www.megunolink.com/documentation/arduino-libraries/exponential-filter/
works perfectly.
